### PR TITLE
fix: onboarding design fixes 

### PR DESCRIPTION
**Pull request by DesmondTambe**
## Description

- Page Swipping animation
- Background color adjustment
- Pointer indicator

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read and ran all relevant commands as specififed in the Running Tests section of the [Contributor Guide].
- [ ] The title of the PR follows the [Conventional Commits] guideline
- [ ] My local branch follows the naming standards in the [Deepsource Branch Naming Convention] or [Biodiversity Branch Naming Convention]
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy],
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

[Contributor Guide]: https://github.com/FlutterPlaza/.github/blob/main/CONTRIBUTING.md
[Conventional Commits]: https://www.conventionalcommits.org/en/v1.0.0-beta.4/
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[Biodiversity Branch Naming Convention]: https://bit.ly/3DyYSwM
[Deepsource Branch Naming Convention]: https://bit.ly/3Y08Gs4

### CODE REVIEW

File: lib/sign_in/view/sign_in_page.dart [hunk 1]
```diff
@@ -221,7 +221,10 @@ class _SignInBodyState extends State<SignInBody>
                                     ),
                                     TextButton(
                                       onPressed: () {
-                                        context.router.push(SignUpRoute());
```
Comment (jeffrey0606) [inline lib/sign_in/view/sign_in_page.dart]: Please revert this changes, It seems like you forget to do so after testing the onboarding

**Review by jeffrey0606 — CHANGES_REQUESTED** [review]

File: lib/sign_in/view/sign_in_page.dart (hunk 1 above)
Comment (DesmondTambe) [reply lib/sign_in/view/sign_in_page.dart]: Oh yeah

Thank you

**Review by jeffrey0606 — APPROVED** [review]
